src/ip/Address.h

```diff
@@ -34,12 +35,41 @@
 namespace Ip
 {
 
+class Address;
+
+class AddressText
+{
+public:
+    explicit AddressText(const Address &ip, bool printPort = false, bool printBrackets = true);
```

Comment:
Please do not allow configuration with anonymous boolean parameters. Most humans cannot reliably remember what AddressText(ip, false, true) means!

```suggestion
    explicit AddressText(const Address &ip);
```

Reply:
done